feature-libs/user/account/components/login-form/login-form.component.html

```diff
@@ -66,14 +66,12 @@
     ></cx-form-errors>
   </label>
 
-  <ng-container *cxFeature="'authorizationCodeFlowByDefault'">
-    <input
-      *ngIf="csrf"
-      formControlName="csrf"
-      type="hidden"
-      [name]="csrf.parameterName"
-    />
-  </ng-container>
+  <input
+    *ngIf="csrf"
+    formControlName="csrf"
+    type="hidden"
+    [name]="csrf.parameterName"
+  />
```

Comment:
The feature flag here isn't necessary because the `csrf` property is only filled when custom login page is enabled.  We could add a property to the component that reflects `customLoginEnabled()` for more explicit logic.